website/content/docs/extending-waypoint/creating-plugins/build-interface.mdx

```diff
@@ -95,14 +95,14 @@ plugin. To do this, you can use `terminal.UI`; this allows you to write output t
 all Waypoint plugins.
 
 The method `ui.Status()` requests a live updating status; you use this to write updates to the Waypoint terminal output.
-When finished with `Status` you should always close a Status, using`st.Close()`; this ensures that the status is updated
+When finished with `Status` you should always close a Status, using`u.Close()`; this ensures that the status is updated
```

Comment:
Since `u` doesn't really exist until the example on 103, can we move the `go` code block above this paragraph in between 95-97? Otherwise this seems a little confusing if you haven't read ahead as to where `u` comes from.